### ISSUE TITLE
Keyboard controls are not working with the main menu

### ISSUE THREAD

**Issue by zipymonkey** [psu-libraries/psulib_base#262]
The main menu dropdown cannot be opened with keyboard controls. 

## Steps to Duplicate
- As an anonymous user
- Go to https://psul-web-profiles-qa.dev.k8s.libraries.psu.edu/
- Using only the keyboard, tab to `Services` main menu item.
- Press the down arrow.
- The sub nav will not open.

## Expected Behavior
You can use https://www.psu.edu as an example of the expected behavior.
- Users should be able to focus on the chevron and open the subnav.
- Chevrons should have helpful text for screen readers
- The `nav_additional` items should halo when focused (this is missing... maybe we want to split this into a separate issue).

**Comment by zipymonkey** [comment]
Adding this to the hershey sprint.  I am not sure if this is also impacting hershey.

**Comment by zipymonkey** [comment]
This is causing a regression in hershey (not seeing this in psul-web) because there are multiple homepage blocks.  

There are some issues with the template and the javascript. There is a hardcoded ID which needs to be removed, plus we need to update the selector to not use the ID.
